### ISSUE TITLE
User media endpoint not working

### ISSUE THREAD

**Issue by daciwei** [SolidlSnake/instagram-go-scraper#3]
Endpoint `https://www.instagram.com/%s/media?max_id=%s` not working anymore.
This is correct endpoint `https://www.instagram.com/{username}/?__a=1&max_id={media_id}` or `https://www.instagram.com/graphql/query/?query_id=17880160963012870&id={{accountId}}&first={count}&after={media_id}`

**Comment by ocabafox** [comment]
@daciwei yeah its because of instagram api changes. Im having same problem to. if only we can curl and get the sessionID of instagram.
